src/main/java/Duke.java

```diff
@@ -1,10 +1,378 @@
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.Scanner;
+import java.util.List;
```

Comment:
I like how you have explicitly listed out the imported classes, adhering to our coding standards 👍